tools/proofers/codepoint_validator.js

```diff
@@ -58,9 +62,67 @@ $(function () {
     });
 
     $("#cc-remove").click(function () {
-        let textarea = $("#text_data");
-        textarea.val(textarea.val().replace(pattern , ""));
+        textArea.value = textArea.value.replace(badPattern , "");
         $("#checker").hide();
         $("#proofdiv").show();
     });
+
+    var above = {
+        "=": "\u0304", // macron
+        ":": "\u0308", // diaeresis
+        ".": "\u0307", // dot
+        "`": "\u0300", // grave
+        "'": "\u0301", // acute
+        "^": "\u0302", // circumflex
+        ")": "\u0306", // breve
+        "~": "\u0303", // tilde
+        ",": "\u0327", // cedilla
+        "v": "\u030C", // caron
+    };
+    var below = {
```

Comment:
We might also consider adding the following (again, extending our current set):
```
        "*": "\u0325", // ring
        "\\": "\u0338", // stoke or bar
        "(": "\u032F", // inverted breve
```

Reply:
I haven't added the stroke like this because it gives a stroke sloping the other way.

Reply:
I'm fine with other symbols, `/` or `-`. Unicode characters WITH STROKE go a few different directions: https://unicode-search.net/unicode-namesearch.pl?term=STROKE

@srjfoo - thoughts?

Reply:
Is this similar to the cedilla vs. the hook issue discussed above, and which code is used? If there's a one-to-one correspondence in our code between the codepoint value and what we use for to stroke type, we probably need different symbols for each one. Otherwise, don't you need a lookup chart for which letters take what kind of stroke, and can't map the symbol the proofer types to just one code value that gets combined with the letter it modifies? (If I'm understanding the issue correctly, and I hope I'm saying it correctly.)

Reply:
There's a one-to-one correspondence on symbol to Unicode diacritical combining character, but how those are represented visually may differ depending on the character. So if `-` maps to STROKE then "capital L with stroke" will have a diagonal slash (Ł) while "capital H with stroke" will have a horizontal slash (Ħ).

IMHO, we should map a symbol to the Unicode diacritical combining character, not how it is visually represented in the glyph, which might also vary by font. That is most straightforward to implement and for people to reference by Unicode name.

Reply:
If the Unicode diacritical combining character is the same, regardless of how it's represented in the glyph, then I don't see a problem with representing the "stroke" with a single character for our purposes. As for `/` fs `-`, which are we more likely to need for a different combining character?

Reply:
I'm not sure stroke would be very useful. Ø can't be composed like this.

Reply:
Can stroke be used to compose the stroked letters in Extended Latin? I'm interested in adding that composite if it's generally useful, even if it's not useful for Basic Latin.

Reply:
I checked the characters in extended latin that have 'stroke' in their description using a tool `https://minaret.info/test/normalize.msp` but none of them would decompose. I tested some with 'bar' and 'hook' which did not decompose either. This seems to imply you can't compose them from combining components either but  I'm not entirely sure about this.
'Ogonek' and 'horn' seem to always decompose so we could perhaps add those.
There are several characters with two diacritical marks. These can be entered as for example [=[~O]] or ['[ae]] or ['[c,]] or we could extend the code to accept e.g. [=~O] as well.

Reply:
So much for consistency in Unicode I guess. Thanks for verifying that. I think we should go with what you have right now. We can always tweak it more later.

Reply:
Sounds good to me, too.

Reply:
I don't want to change it any more now. As Casey says, we can tweak it later.